lib/migration.pm

```diff
@@ -150,8 +150,8 @@ sub remove_dropped_modules_packages {
 # https://documentation.suse.com/sles/15-SP2/html/SLES-all/cha-upgrade-online.html#sec-upgrade-online-zypper
 sub disable_installation_repos {
     if (is_s390x) {
-        my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}'");
-        zypper_call "mr -d $repos" if (length $repos);
+        my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}' | tr '\n' ' '");
+        zypper_call "mr -d $repos";
```

Comment:
Please reserve the condition of "if (length $repos) " which used to ensure the repos is not a NULL.